evergreen/Dockerfile

```diff
@@ -21,9 +21,8 @@ RUN chmod +x /wait
 RUN ["/wait"]
 
 SHELL [ "poetry", "run" ]
+CMD [ "python3","manage.py","runserver","0.0.0.0:8080"]

```

Comment:
idk why this was moved but the file is outdated anyway. just bring this back down to remove this file from the PR and avoid a conflict